processor/processor.go

```diff
@@ -2573,3 +2607,72 @@ func filterConfig(eventCopy *transformer.TransformerEventT) {
 		}
 	}
 }
+
+func (proc *Handle) filterDestinations(
+	event types.SingularEventT,
+	dests []backendconfig.DestinationT,
+) []backendconfig.DestinationT {
+	eventOneTrustCookies, _ := misc.MapLookup(
+		event,
+		"context",
+		"consentManagement",
+		"deniedConsentIds",
+	).([]interface{})
+	if len(eventOneTrustCookies) == 0 {
+		return dests
+	}
+	eventCookies := lo.FilterMap(
+		eventOneTrustCookies,
+		func(cookie interface{}, _ int) (string, bool) {
+			cookieStr, ok := cookie.(string)
+			return cookieStr, ok
+		},
+	)
```

Comment:
doing this instead(in `processor` package), because `types` is already imported in `misc`
```
func deniedConsentCategories(se types.SingularEvent) []string {
	if deniedConsents, _ := misc.MapLookup(se, "context", "consentManagement", "deniedConsentIds").([]interface{}); len(deniedConsents) > 0 {
		return lo.FilterMap(
			deniedConsents,
			func(consent interface{}, _ int) (string, bool) {
				consentStr, ok := consent.(string)
				return consentStr, ok && consentStr != ""
			},
		)
	}
	return nil
}
```